### PR TITLE
Add PGrnExprAppendOp()

### PR DESCRIPTION
**Pull request by abetomo**
Check with PGrnCheck() after grn_expr_append_op().

### CODE REVIEW

**Comment by abetomo** [comment]
We will replace `grn_expr_append_op()` after this PR.

File: src/pgrn-groonga.c [hunk 1]
```diff
@@ -655,3 +655,11 @@ PGrnGrnTypeToPGType(grn_id typeID)
 
 	return pgTypeID;
 }
+
+void
+PGrnExprAppendOp(grn_obj *expr, grn_operator op, int nargs, const char *tag)
```
Comment (kou) [inline src/pgrn-groonga.c:660]:
```suggestion
PGrnExprAppendOp(grn_obj *expr, grn_operator op, int nArgs, const char *tag)
```

File: src/pgrn-groonga.c [hunk 2]
```diff
@@ -655,3 +655,11 @@ PGrnGrnTypeToPGType(grn_id typeID)
...
+{
+	grn_expr_append_op(ctx, expr, op, nargs);
```
Comment (kou) [inline src/pgrn-groonga.c:662]:
```suggestion
	grn_expr_append_op(ctx, expr, op, nArgs);
```

File: src/pgrn-groonga.c [hunk 3]
```diff
@@ -655,3 +655,11 @@ PGrnGrnTypeToPGType(grn_id typeID)
...
+	PGrnCheck(
+		"%s: failed to append operator: %s", tag, grn_operator_to_string(op));
```
Comment (kou) [inline src/pgrn-groonga.c:664]:
```suggestion
		"%s: failed to append operator: %s(%d)", tag, grn_operator_to_string(op), nArgs);
```

File: src/pgrn-groonga.h [hunk 4]
```diff
@@ -371,3 +371,5 @@ PGrnRegisterPluginWithSize(const char *name, size_t nameSize, const char *tag)
 
 grn_id PGrnPGTypeToGrnType(Oid pgTypeID, unsigned char *flags);
 Oid PGrnGrnTypeToPGType(grn_id typeID);
+void
+PGrnExprAppendOp(grn_obj *expr, grn_operator op, int nargs, const char *tag);
```
Comment (kou) [inline src/pgrn-groonga.h:375]:
```suggestion
PGrnExprAppendOp(grn_obj *expr, grn_operator op, int nArgs, const char *tag);
```

**Pull request merged**